src/components/BaseButton.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+ <button type="button" :class="[size, backgroundColor]" >Trimite
```

Comment:
Not sure why there is text there

Reply:
Also a better naming would be just color rather than backgoundcolor

Reply:
```
<button
      :class="[size, backgroundColor]"
      v-bind="$attrs"
      v-on="$listeners"
 >
    <slot />
 </button>
```
This would be a better approach you don't have to specify every atribute or event